src/widget/wspinny.h

```diff
@@ -69,12 +70,16 @@ class WSpinny : public QGLWidget, public WBaseWidget, public VinylSignalQualityL
   private:
     QString m_group;
     UserSettingsPointer m_pConfig;
-    QImage* m_pBgImage;
-    QImage* m_pMaskImage;
-    QImage* m_pFgImage;
-    QImage m_fgImageScaled;
-    QImage* m_pGhostImage;
-    QImage m_ghostImageScaled;
+
+    QPixmap m_bgPixmap;
```

Comment:
There is some history for why WSpinny uses QImage.

https://bugs.launchpad.net/mixxx/+bug/981210
https://github.com/mixxxdj/mixxx/commit/a15c2687db5bf3c3b0cdf48ad815f6eeec1ac9dd
https://github.com/mixxxdj/mixxx/pull/440

@daschuer -- do you still have this netbook? I wonder if this bug in Qt 4.6.2 was fixed in a later version.